gitlab/__init__.py

```diff
@@ -568,7 +568,7 @@ def update(self, obj, **kwargs):
         params.update(kwargs)
         missing = []
         for k in itertools.chain(obj.requiredUrlAttrs,
-                                 obj.requiredCreateAttrs):
+                                 obj.requiredUpdateAttrs):
```

Comment:
Thank you for explanation. As I see it now that change also requires to specify correct requiredUpdateAttrs for all GitlabObjects. Please correct me if that is not in sync with changes since 9.0.2 to address fixme on User object.